pkg/cmd/server/origin/auth_config.go

```diff
@@ -72,6 +73,7 @@ func BuildAuthConfig(options configapi.MasterConfig) (*AuthConfig, error) {
 	ret := &AuthConfig{
 		Options: *options.OAuthConfig,
 
+		WebConsoleDisabled:   options.DisabledFeatures.Has(configapi.FeatureWebConsole),
```

Comment:
remove this... the only difference the web console being disabled should make to auth is that assetPublicURLs be empty